gsl_simulation/static/js/handleInteractionsInSimulation.js

```diff
@@ -23,3 +23,12 @@ document.querySelector(".gsl-projet-table").addEventListener("change", (ev) => {
     }
     return handleStatusChangeWithHtmx(target, target.dataset.originalValue);
 })
+
+document.addEventListener('htmx:responseError', evt => {
+    const xhr = evt.detail.xhr;
+  
+    if (xhr.status == 400) {
```

Comment:
Que dis-tu d'ajouter de la robustesse sur d'autres possibilités d'erreur ?  404, 500, etc.

```suggestion
    if (xhr.status >= 400) {
```